server/src/test/java/org/elasticsearch/search/aggregations/bucket/terms/StringTermsIT.java

```diff
@@ -87,7 +87,7 @@
 
     @Before
     public void randomizeOptimizations() {
-        TermsAggregatorFactory.COLLECT_SEGMENT_ORDS = false;randomBoolean();
+        TermsAggregatorFactory.COLLECT_SEGMENT_ORDS = randomBoolean();
```

Comment:
I also opened a separate PR to separate this change: https://github.com/elastic/elasticsearch/pull/33678

Reply:
Actually it wasn't reformatting, just my Auto-Save trailing whitespace removal.